client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -168,6 +171,15 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
     onDeleteTaskgroupError
   );
 
+  const useWizardValidationSchema = () => {
```

Comment:
This doesn't technically need to have the `use` naming since it's not calling any hooks, the `yup.object()` could just be inlined inside the `yupResolver()` call. But we can leave it like this since it probably makes sense to move this out of this file as part of refactoring the wizard form state anyway, and it'll need to be a hook for that so it can call `const { t } = useTranslation();`